src/lib/phdi-building-blocks/phdi_building_blocks/fhir.py

```diff
@@ -14,50 +14,45 @@
 from azure.storage.blob import download_blob_from_url
 
 
-def generate_filename(blob_name: str, message_index: int) -> str:
-    """Strip the file type suffix from the blob name, and append message index."""
-    fname = blob_name.split("/")[-1]
-    fname, _ = fname.rsplit(".", 1)
-    return f"{fname}-{message_index}"
-
-
-class AzureFhirserverCredentialManager:
-    """Manager for handling Azure credentials for access to the FHIR server"""
+class AzureFhirServerCredentialManager:
```

Comment:
This whole class should be moved to what is currently named `azure_blob.py` and the file should be renamed to `azure.py`